src/one-time-token/formats/hal.ts

```diff
@@ -0,0 +1,9 @@
+export function oneTimeToken(url: URL) {
+  return {
+    _links: {
+      self: {
+        href: url
```

Comment:
self should refer to the URL of the resource, so whatever url was used to access the endpoint.

Make a new link for the lost-password link.

In a previous version you also had the token itself in the body, and i think that that might still be helpful!